### PR TITLE
WIP: Merge runner autostart

### PR DESCRIPTION
**Pull request by agaida**
Please do not merge now

### CODE REVIEW

File: autostart/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+file(GLOB DESKTOP_FILES *.desktop)
+
+
+add_custom_target(autostart_desktop_files ALL DEPENDS ${DESKTOP_FILES})
+#************************************************
+
+install(FILES
+    ${DESKTOP_FILES}
+    DESTINATION "${LXQT_ETC_XDG_DIR}/autostart"
+    COMPONENT Runtime
```
Comment (palinek) [inline autostart/CMakeLists.txt]: How about using some other COMPONENT? @luis-pereira ?

Comment (luis-pereira) [inline autostart/CMakeLists.txt]: @palinek I'm terribly lost here. What's this about ?

Comment (palinek) [inline autostart/CMakeLists.txt]: These are the "autostart" .desktop files for LXQt apps/daemons... so just thought they can be handled by cmake as a different COMPONENT... not the `Runtime` one

Comment (luis-pereira) [inline autostart/CMakeLists.txt]: Ok.
I splitted files into two componets:
* Development - for files only needed to build the package
* Runtime - for files needed at runtime, another way of saying everything else. 

What component do you have in mind ?

Comment (palinek) [inline autostart/CMakeLists.txt]: I don't know... Data ? It was just an idea... but the Runtime is also OK.

Comment (luis-pereira) [inline autostart/CMakeLists.txt]: If that's fine from a packaging perspective, I'm Ok with it. @agaida as certainly a more informed opinion.

Comment (agaida) [inline autostart/CMakeLists.txt]: from a packaging perspective i see no problems

**Pull request closed without merging**